easybuild/easyconfigs/g/GCC/GCC-4.9.3-binutils-2.25.eb

```diff
@@ -0,0 +1,45 @@
+name = "GCC"
+version = '4.9.3'
+
+binutilsver = '2.25'
+versionsuffix = '-binutils-%s' % binutilsver
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.2'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/%(namelower)s/%(namelower)s-%(version)s',  # GCC auto-resolving HTTP mirror
+    'http://ftpmirror.gnu.org/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+]
+sources = [
+    SOURCELOWER_TAR_BZ2,
+    'gmp-6.0.0a.tar.bz2',
+    'mpfr-%s.tar.gz' % mpfr_version,
+    'mpc-1.0.2.tar.gz',
+]
+
+builddependencies = [('binutils', binutilsver)]
+
+patches = [('mpfr-%s-allpatches-20141204.patch' % mpfr_version, '../mpfr-%s' % mpfr_version)]
```

Comment:
doesn't this require bumping the MPFR version too (which we won't here, to keep this in sync with the existing easyconfig for GCC 4.9.3)? the latest `allpatches` requires the latest MPFR version?